Practice-two/src/assets/styles/abstracts/variables.css

```diff
@@ -0,0 +1,6 @@
+:root {
+    --primary-color: #02897A;
+    --text-color: #22343D;
```

Comment:
you should specify what color it is, not `text-color`. For example: red, white, blue,...